libminifi/include/utils/StringUtils.h

```diff
@@ -159,9 +160,9 @@ class StringUtils {
   }
 
   inline static std::string hex_ascii(const std::string& in) {
-    int len = in.length();
+    size_t len = in.length();
```

Comment:
I think we can simply remove the variable, it's only referenced in the header of the loop. 

Completely unrelated, possible optimisation here: guess the lenght of the output and reserve place for the result string with that size to avoid reallocations caused by push_back.

Reply:
I have inlined the variable, and added a `reserve` (though I don't think this will increase performance by a measurable amount).